pkg/controllers/image/image.go

```diff
@@ -278,6 +287,32 @@ func (h handler) onChangeGitRepo(gitrepo *v1alpha1.GitRepo, status v1alpha1.GitR
 	return status, err
 }
 
+func (h handler) setupKnownHosts(gitrepo *v1alpha1.GitRepo, data []byte) error {
+	tmpdir, err := os.MkdirTemp("", fmt.Sprintf("ssh-%s-%s-", gitrepo.Namespace, gitrepo.Name))
```

Comment:
```suggestion
	tmpdir, err := os.MkdirTemp("", fmt.Sprintf("fleet-ssh-%s-%s-", gitrepo.Namespace, gitrepo.Name))
```

So we can easily remove the dirs locally, without interfering with other temp files?